orb-ui/src/sound/mod.rs

```diff
@@ -238,8 +252,19 @@ sound_enum! {
     }
 }
 
+/// Default sound volume
+const DEFAULT_SOUND_VOLUME_PERCENT: u64 = 10;
+
+pub struct Jetson {
+    _stream_task: StreamTask,
+    _stream_handle: rodio::OutputStreamHandle,
+    queue_file: Mutex<mpsc::Sender<PathBuf>>,
```

Comment:
a `mpsc` implements `Sync` and also can be cloned. You should be able to just get a &mut to it fine without a mutex. However, your other methods on `Jetson` will need to take `&mut Self` 

Thats fine, because the jetson is designed to be a cloneable, `Sync` proxy object. So you can be sure that you are gonna have exclusive access to self - if it needed to be shared, the user would just clone it.